frontend/src/components/Blog/Story.tsx

```diff
@@ -18,11 +18,16 @@ import { formatDateString } from '../../util/string';
 import VoiceOver from '../VoiceOver';
 import { getPlainTextFromHTML } from '../../util/string';
 import ChatModule from '../ChatModule';
+import { useRecoilValueLoadable } from 'recoil';
+import { singlePost } from '../../store/atoms/singlePost';
+import { Post } from '../../types/post';
 
 const Story = () => {
   const { id } = useParams();
   const navigate = useNavigate();
-  const { blog, loading } = useBlog({ id: id || '' });
+  const singlePosttt = useRecoilValueLoadable(singlePost(id));
+  const loading = singlePosttt.state==="loading";
+  const blog:Post = singlePosttt.contents;
```

Comment:
It would be cleaner to implement this inside useBlog and let useBlog return the blog, loading the way it originally did.

Reply:
hmm lemme use cleaner format